sql/codegen_alps.go

```diff
@@ -539,7 +539,7 @@ if __name__ == "__main__":
         run_experiment(experiment)
     else:
         if "{{.ExitOnSubmit}}" == "false":
-            run_experiment(experiment)
+            submit_experiment(experiment)
```

Comment:
Just feeling confusion: does `submit_experiment` equal to `run_expriment`?

Reply:
just a bug in alps.